invenio/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.persistence.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Values.persistence.name }}
+  labels:
+    app: web
+spec:
+  storageClassName: {{ .Values.persistence.storage_class }}
```

Comment:
@wgresshoff fixed! Thanks for the review. I checked and if the default value of the variables is `""` then it takes the default storge class configured in the cluster, so we are good :) no need for `if`'s